packages/integration-sdk-cli/src/neo4j/wipeNeo4j.ts

```diff
@@ -0,0 +1,43 @@
+import { Neo4jGraphStore } from './neo4jGraphStore';
+import * as log from '../log';
+
+export async function wipeNeo4jByID(integrationInstanceID: string) {
+  if(!process.env.NEO4J_URI || !process.env.NEO4J_USER || !process.env.NEO4J_PASSWORD) {
+    throw new Error('ERROR: must include NEO4J_URI, NEO4J_USER, and NEO4J_PASSWORD files in your .env file!');
+  }
+  log.info(`Wiping Neo4j data for integrationInstanceID: ${integrationInstanceID}`);
```

Comment:
I generally try to avoid logging in utility functions like this. I feel that it's the job of the consumer to log and try to keep functions like this as pure as possible.

Reply:
Updated in latest commit.